typescript/common-resolvers/src/types.ts

```diff
@@ -139,9 +140,12 @@ export type Repository = {
     update: Update<DbDataset, DatasetWhereUniqueInput>;
   };
   workspace: {
-    add: Add<WorkspaceCreateInput>;
+    add: Add<WorkspaceCreateInput & { slug: string }>;
```

Comment:
~~That does not look right at first.~~

~~Why do you not add `slug: string` to `WorkspaceCreateInput` directly instead ?~~

~~It seems weird to me that the resolver does not accept exactly the input type.~~

~~Also, being able to specify `slug` and `name` separately in the GraphQL query is a security problem. The name => slug conversion should be done inside the graphql resolver, otherwise user can abuse the API to create workspaces where slug does not match name.~~

Reply:
Ok got it, you moved this, not to the client, but to the resolver, to commonalize it. Good